cmd/podman/machine/cp.go

```diff
@@ -0,0 +1,137 @@
+//go:build amd64 || arm64
+
+package machine
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"strconv"
+
+	"github.com/containers/common/pkg/completion"
+	"github.com/containers/podman/v5/cmd/podman/registry"
+	"github.com/containers/podman/v5/libpod/events"
+	"github.com/containers/podman/v5/pkg/copy"
+	"github.com/containers/podman/v5/pkg/machine"
+	"github.com/containers/podman/v5/pkg/machine/define"
+	"github.com/containers/podman/v5/pkg/machine/env"
+	"github.com/containers/podman/v5/pkg/machine/vmconfigs"
+	"github.com/spf13/cobra"
+)
+
+var (
+	cpCmd = &cobra.Command{
+		Use:               "cp [options] [MACHINE:]SRC_PATH [MACHINE:]DEST_PATH",
```

Comment:
This doesn't look right, this means that we're allowing copying from one machine to another machine, which shouldn't be the case.

Reply:
I'm not sure how to show that you can go from host -> guest or guest -> host in a single line. Do you have any suggestions?

Reply:
```suggestion
		Use:               "cp [options] SRC_PATH DEST_PATH",
		Short:             "Securely copy contents between the virtual machine",
		Long:              "Securely copy files or directories between the virtual machine and your host",
		PersistentPreRunE: machinePreRunE,
		RunE:              cp,
		Args:              cobra.ExactArgs(2),
		Example:           `podman machine cp myfile machine-name:/tmp/myfile`
podman machine cp machine-name:/tmp/myfile myfile,
```

Something like this?  And then update the manpage as well?

Reply:
I'm ok with that. I do wish there was an easier way to show the notation for giving the machine name...

Reply:
Fixed!

Reply:
Tbh I think the `[MACHINE:]SRC_PATH [MACHINE:]DEST_PATH` is more logical, it matched podman cp as well.